### ISSUE TITLE
[Bug] Please uprade openvino-dev deps

### ISSUE THREAD

**Issue by luohao123** [openvinotoolkit/openvino#12888]
<img width="606" alt="image" src="https://user-images.githubusercontent.com/49749220/188294472-749a3f76-76a4-4e86-a239-be6b4ed0d3fa.png">

I dont really understand why this.

**Comment by eaidova** [comment]
@luohao123 from your logs it is difficult to understand what went wrong and why you want upgrade dependencies... 
Could you please provide more details?

I see only following, you have in environment transformers 3.0.12, but openvino-dev inside requires 4.5 and above. Which python version do you use? Is it important for you to use too old transformers version  (I do not see it on pypi, probably it is installed from source, but it was in begining 2020 year...)?

**Comment by luohao123** [comment]
No, I didn't have transformers installed before install openvino-dev. It was because openvino-dev requries transformers >= 4.15, but the transformers has newest 4.21.... So it will downl;oad all possible version and choose one.... You should better require more strict version requirements...

**Comment by eaidova** [comment]
@luohao123 which pip version do you have?
Let me check, as I know it should work slightly different:
in normal way, pip tries to search dependencies with the best match conditions (so 4.21 >= 4.15 right? and should be ok) and then download wheel which satisfy, it should not install all, that is why your logs looks strange for me...

**Comment by luohao123** [comment]
@eaidova Yes, also strange to me, but, it downloaded all, cost too many time and too many data usage of my network.

**Comment by eaidova** [comment]
@luohao123  that is why I ask to provide your pip and python version, I will try to reproduce locally

**Comment by luohao123** [comment]
I think my pip install lastest (I runed pip install --upgrade pip). python 3.8

**Comment by andrei-kochin** [comment]
@luohao123 will it be possible for you to provide full logs as a file?

Also please provide the pip version as @eaidova  asked

**Comment by helena-intel** [comment]
I can also reproduce this with Python 3.8, latest pip (22.2.2), Ubuntu 20.04. The solution is not to use more strict version requirements, because OpenVINO should work fine with multiple transformers versions and if a user needs, say, 4.16 it is needlessly restrictive if we pin to 4.15. 

The problem seems to be that openvino-dev requires a pretty strict tokenizers version (~=0.10.1) and pip tries to find a transformers version that satisfies that requirement. If we can relax the version pinning of tokenizers a bit, this issue should be resolved, and openvino-dev will have better compatibility with transformers. For example, this is the error you get if you install transformers 4.19 in an environment with openvino-dev: `transformers 4.19.0 requires tokenizers!=0.11.3,<0.13,>=0.11.1, but you have tokenizers 0.10.3 which is incompatible`.

**Comment by helena-intel** [comment]
This should already be solved for Python 3.8 in the new prerelease of OpenVINO 2022.2. @luohao123 if you do `pip install --pre openvino openvino-dev` you should not see this issue anymore.

**Comment by eaidova** [comment]
You are right, I already fixed it 6 months ago :)
https://github.com/openvinotoolkit/open_model_zoo/pull/3357

**Comment by luohao123** [comment]
@helena-intel hi, I didn't used the `--pre` here. that's mgith be the problem

**Comment by helena-intel** [comment]
You're not expected to use `--pre` so this was not something you could have known. But the fact that the issue does not occur in the 2022.2 prerelease means that it is already fixed from our side. It can sometimes take a while before these fixes make it to the pip releases. The final release of 2022.2 should be out soon and the issue should not occur there. Unfortunately Python dependencies can be tricky sometimes. Thank you for reporting the issue!

**Comment by avitial** [comment]
Closing this as it has been addressed in v2022.2 release. Feel free to reopen and ask any questions related to this topic.
